Fix bug squid installation on stratum1

ahmam opened the pull request:
Add variable to make choose  to configure squid in stratum 1 or not.

ahmam committed:
Add squid package for default installation stratum 1 on debian

Add squid package for default installation stratum 1 on debian

ahmam committed:
Add variable cvmfs_strqtum1_squid

Add variable "cvmfs_strqtum1_squid" to make choose  to install squid in stratum 1 or not

ahmam committed:
Add variable to make choose  to install squid in stratum 1 or not

Add variable to make choose  to install squid in stratum 1 or not

btravouillon commented:
Has this bug been reported to upstream project and acknowledged?

milaflq reviewed (commented):
@ahmam
[Add variable cvmfs_strqtum1_squid](https://github.com/mila-iqia/ansible-cvmfs/pull/2/commits/54b87be7c0769ab015c2ab700cea53f13afaed2b)

Est-ce un typo? (cvmfs_strqtum1_squid)

milaflq commented on defaults/main.yml:
```diff
@@ -20,6 +20,8 @@ cvmfs_localproxy_http_ports:
 
 cvmfs_stratum1_apache_port: 8008
 cvmfs_stratum1_cache_mem: 128 # MB
+# Automatically configure a squid on startum1.
```
"startum1"
Typo?

ahmam replied:
oui c'est type je corrige

milaflq reviewed (approved):
Sinon, le code est bon de mon point de vue.

ahmam committed:
fix typo

ahmam commented:
> Has this bug been reported to upstream project and acknowledged?

it is not reported to upstream because it is really optional it is a feature that we need but I will report it in a pull request.

btravouillon commented:
> it is not reported to upstream because it is really optional it is a feature that we need but I will report it in a pull request.

Looks like package `squid` is also added in upstream PR that you mentionned in #3. Looking further, the `squid` package is installed on the stratum1 without condition for RHEL-like hosts (https://github.com/galaxyproject/ansible-cvmfs/blob/0.2.21/vars/redhat.yml#L19). Thus I believe you can just add the package, you don't have to introduce any new variable that would change the behaviour of the role for RHEL-like hosts.

ahmam committed:
remove the variable

ahmam committed:
delete the added variable

ahmam committed:
Merge branch 'mila' into fix-bug-squid-installtion-on-stratum1

ahmam commented:
> > it is not reported to upstream because it is really optional it is a feature that we need but I will report it in a pull request.
> 
> Looks like package `squid` is also added in upstream PR that you mentionned in #3. Looking further, the `squid` package is installed on the stratum1 without condition for RHEL-like hosts (https://github.com/galaxyproject/ansible-cvmfs/blob/0.2.21/vars/redhat.yml#L19). Thus I believe you can just add the package, you don't have to introduce any new variable that would change the behaviour of the role for RHEL-like hosts.

I just added the package  squid to comply with redhat and remove the changes made by the variable.

btravouillon reviewed (approved):
Looks good to me. 👍
